### ISSUE TITLE
html format

### ISSUE THREAD

**Issue by GerhardRiphagen** [stevermeister/ngWig#39]
When you look at text in html, it's not very convenient to read. Indention on html would be convenient.

**Comment by bampakoa** [comment]
@stevermeister I think its a good case for another plugin. What do you think?

**Comment by stevermeister** [comment]
@bampakoa it's awesome task, I was looking for easy solution for it, but in fact solution of this issue if we take in account all the cases is pretty complex

**Comment by bampakoa** [comment]
@stevermeister I agree that is too complex to implement it from scratch and there is no point to re-invent the wheel. I was thinking of creating a plugin in conjunction with the use of a custom library (e.g. [js-beautify](https://github.com/beautify-web/js-beautify)).Sure, there will be more like this. But this is what came to my mind at once :smile:

**Comment by bampakoa** [comment]
@stevermeister any news on the above proposal?
